parcellab/common/values.yaml

```diff
@@ -20,6 +20,7 @@ image:
 ingress:
   enabled: false
 name: common
+terminationGracePeriodSeconds: 30
```

Comment:
can we add this as placeholder to the workers example in `monolith` and `worker-group` too, as this is quite an important setting for those types of deployments. I assume that this option also works for those types, right?